src/body.js

```diff
@@ -231,6 +232,10 @@ function consumeBody(body) {
  * @return  String
  */
 function convertBody(buffer, headers) {
+	if (typeof convert !== 'function') {
+		throw new FetchError('The package `encoding` must be installed to use the textConverted() function', 'missing-encoding');
```

Comment:
I think Error is better here as it's a programming error instead of operational error.

https://github.com/bitinn/node-fetch/blob/master/ERROR-HANDLING.md